src/+datools/+statistical/+ensemble/RHF.m

```diff
@@ -1,158 +1,167 @@
 classdef RHF < datools.statistical.ensemble.EnF
-    
+
     properties
         Tail
         Truncate
         DiscretePoint
     end
-    
-    % Define the update method
+
     methods
         function obj = RHF(varargin)
             p = inputParser;
             p.KeepUnmatched = true;
-            
+
             addParameter(p, 'Tail', 'Flat');
             addParameter(p, 'Truncate', 10);
             addParameter(p, 'DiscretePoint', 1e4);
             parse(p, varargin{2:end});
-            
+
             s = p.Results;
-            
+
             kept = p.Unmatched;
-            
+
             obj@datools.statistical.ensemble.EnF(varargin{1}, kept);
             obj.Tail = s.Tail;
             obj.Truncate = s.Truncate;
             obj.DiscretePoint = s.DiscretePoint;
         end
-        
+
         function analysis(obj, R, y)
             inflation = obj.Inflation;
             tc = obj.Model.TimeSpan(1);
-            
-            % define length of gaussian tail
-            gtl = obj.Truncate;
-            discrete_pts = obj.DiscretePoint;
-            
+
+            % define length of gaussian tail (not needed now)
+            %gtl = obj.Truncate;
+            %discrete_pts = obj.DiscretePoint;
```

Comment:
If it's not needed, why is it still here. delete

Reply:
Deleted